README.md

```diff
@@ -1 +1,60 @@
-# Todo-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo-list
+
+> Description the project.
+> A todo list using webpack
+
+## Built With
+
+- Html
+- CSS
+- JS
+- Webpack
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+- ESLint
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+The basic requirements for building the executable are:
+* A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+* VSCode or any other equivalent code editor
+* Node Package Manager (For installing packages like Lighthouse, webhint, stylelint & eslint used for checking for debugging bad codes before deployment)
+
+## Getting Started
+
+#### Cloning the project
+```
+git clone https://github.com/Yorsyboy/Todo-list.git
+```
+
```

Comment:
- [OPTIONAL] It will be nice to add explicit steps to on how to run your app locally as this will be of help to non technical users.